pkgs/tools/networking/dropbear/default.nix

```diff
@@ -1,8 +1,27 @@
-{ stdenv, lib, fetchurl, glibc, zlib
+{ stdenv
+, lib
+, fetchurl
+, glibc
+, zlib
 , enableStatic ? stdenv.hostPlatform.isStatic
```

Comment:
wouldn't it be a better idea to put this support into [`pkgs/top-level/static.nix`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/static.nix)?